client/convert/protobuf.go

```diff
@@ -65,10 +66,11 @@ func MessageToAccount(m *entities.Account) (flow.Account, error) {
 	}
 
 	return flow.Account{
-		Address: flow.BytesToAddress(m.GetAddress()),
-		Balance: m.GetBalance(),
-		Code:    m.GetCode(),
-		Keys:    accountKeys,
+		Address:   flow.BytesToAddress(m.GetAddress()),
+		Balance:   m.GetBalance(),
+		Code:      m.GetCode(),
+		Keys:      accountKeys,
+		Contracts: m.GetContracts(),
```

Comment:
Seeing this again, I'm now wondering if the contract ordering matters in the protobuf response, too. 

The order of keys in a serialized protobuf map is undefined, so we can't be sure that this `Contracts` field will always be returned in the same order. But is that okay?

Reply:
When deploying the contracts order matters to have a consistent state commitment. When getting the contracts I don't se a reason why the order should be deterministic. How does the order matter to the user?

On the other hand its slightly weird that you put an array in and get out a map...

Reply:
Hmm, it shouldn't matter, should it? I guess if one wanted to reconstruct, one couldn't. 
Given that the `contracts` field is in the protobuf, but was only just added, we can still change it easily?